src/components/common/AnswerComponent/AnswerComponent.jsx

```diff
@@ -0,0 +1,25 @@
+export const AnswerComponent = ({
+  mainWord,
+  randomWords,
+	setFeedback,
+	correctFeedback,
+	incorrectFeedback
+})	=> {
+	const handleAnswer = (wordObject) =>{
+		if(wordObject.value === mainWord)	{
+			setFeedback(correctFeedback);
+		}	else {
+			setFeedback(incorrectFeedback);
+		}
+	};
+
+	return (
+		<div>
+			{randomWords.map((wordObject, index)	=>	(
+				<button key={index} onClick={() => handleAnswer(wordObject)}>
+					{wordObject.value}
+				</button>
+			))}
+		</div>
+	);
```

Comment:
Too large spaces, something wrong with alignment. Have you run eslint and prettier?